### DISCUSSION TITLE
Implement unity like top panel to save screen space

### DISCUSSION THREAD

**Discussion by EmmanouelKontos** [elementary/triage#375] [Ideas]
First of all I want to say to the developers and the rest of the team they are doing very good job ! 
Now I want to suggest unity like top panel. I think is really cool when you maximize a window the top bar of window integrates with top panel of OS. Like the picture. Plus you save space.
![screenshot_1287_IwQGNU5](https://github.com/elementary/triage/assets/58360908/c472cf74-b9e5-4030-8fcc-183e183e88cc)

**Comment by secondlnl** [comment]
I think that wouldn't work as the buttons for closing and maximizing would get in the way of the Applications menu.

**Reply by fayjies** [reply]
Another way to achieve this is by adding a setting for users to autohide the top panel.
